server/settings/settings.proto

```diff
@@ -32,16 +32,17 @@ message Settings {
     repeated github.com.argoproj.argo_cd.v2.pkg.apis.application.v1alpha1.ConfigManagementPlugin configManagementPlugins = 12;
     repeated string kustomizeVersions = 13;
     string uiCssURL = 14;
-    string uiBannerContent = 15;
-    string uiBannerURL = 16;
-    string passwordPattern = 17;
-    string trackingMethod = 18;
-    bool uiBannerPermanent = 19;
-    string uiBannerPosition = 20;
-    string statusBadgeRootUrl = 21;
-    bool execEnabled = 22;
-    string controllerNamespace = 23;
-    bool appsInAnyNamespaceEnabled = 24;
+    string UiTitle = 15;
```

Comment:
Changing the numbers of existing fields in the `proto` file will break backward compatibility. Please append the `uiTitle` field at the end of this list as `string uiTitle = 25;`

Additionally, the convention is to lowercase `ui`, so make sure it is lowercase for the new field as well

Reply:
yep, typo, fixed